Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,58 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+User Applications can make use of the NFS client running on the external ceph standalone cluster.
```

Comment:
we already have NFS CSI driver why users need to create static pv/pvc to access it from intree code?

Reply:
Yes we had two ways, the first one looked more direct.

So if the user already has the nfs exports, they can use this document to create the pv and pvcs https://docs.openshift.com/container-platform/4.14/storage/persistent_storage/persistent-storage-nfs.html
       Moreover if they don't have the exports created they can run the simple ceph cmd
      ceph nfs export create cephfs my-nfs /test myfs 
2. Secondly we can integrate this feature similar way we have other addons,
   Steps would be creating a new nfs-storage class from the python script and then ceph-csi will take care of the export and create the pv for us. (and need  update on the csi-users permissions)

Reply:
NFS is not deprecated, but kubernetes already have the official doc for it https://github.com/kubernetes/examples/tree/master/staging/volumes/nfs 
https://kubernetes.io/docs/concepts/storage/volumes/#nfs
why do we need to reimplement it?

Reply:
That's correct I can add the link to that content, thanks!
These are the examples listed,
https://github.com/kubernetes/examples/tree/master/staging/volumes/nfs

Reply:
That's correct I can add the link to that content, thanks!
These are the examples listed,
https://github.com/kubernetes/examples/tree/master/staging/volumes/nfs

Reply:
Updated, how about now